src/presentation/ui/page/marketplace.templ

```diff
@@ -0,0 +1,391 @@
+			for _, installedItem := range installedItemsList {
+				<tr class="odd:bg-os-300 even:bg-os-500 border-b border-neutral-500 border-opacity-30 text-center">
+					<th scope="row" class="w-1/6 px-5 py-3 text-left font-normal">
+						<div class="inline-flex items-center">
+							<img class="mr-2 h-10 w-10 rounded-full" src={ installedItem.AvatarUrl.String() }/>
+							{ installedItem.Name.String() }
+						</div>
+					</th>
+					<th scope="row" class="w-1/6 px-5 py-3 font-normal">{ installedItem.Type.String() }</th>
+					<th scope="row" class="w-1/6 px-5 py-3 font-normal">
+						{ installedItem.Hostname.String() + installedItem.UrlPath.String() }
+					</th>
+					<th scope="row" class="w-1/6 px-5 py-3 font-normal">{ installedItem.InstallDirectory.String() }</th>
+					<th scope="row" class="w-1/6 px-5 py-3 font-normal">{ installedItem.CreatedAt.GetDateOnly() }</th>
+					<th scope="row" class="w-1/6 px-5 py-3 text-right font-normal">
+						<div class="inline-flex">
+							@componentStructural.CircularIconButtonWithTooltip(
+								"ph-trash", "red-800", "red-600",
+								"openUninstallMarketplaceItemModal("+installedItem.Id.String()+", '"+installedItem.Name.String()+"')",
+								"Uninstall", "red-500",
+							)
+						</div>
+					</th>
+				</tr>
+			}
+		</tbody>
+	</table>
+}
+
+templ MarketplaceCatalogItemCarouselImage(
+	itemFirstSlug valueObject.MarketplaceItemSlug,
+	itemName valueObject.MarketplaceItemName,
+	screenshotUrl valueObject.Url,
+) {
+	<div x-data="{ isImageLightboxOpen: false }">
+		<div
+			x-show="isImageLightboxOpen"
+			x-on:click.away="isImageLightboxOpen = false"
+			class="fixed inset-0 z-50 flex h-full w-full items-center justify-center overflow-hidden bg-black bg-opacity-75 p-2"
+		>
+			<div @click.away="isImageLightboxOpen = false" class="flex flex-col overflow-auto">
+				<div class="z-50">
+					<button
+						@click="isImageLightboxOpen = false"
+						class="opacity-85 float-right bg-transparent pr-2 pt-2 hover:opacity-100 active:opacity-100"
+					>
+						<i class="ph-bold ph-x-square align-middle text-3xl"></i>
+					</button>
+				</div>
+				<div class="p-2">
+					<img
+						class="max-h-150"
+						src={ screenshotUrl.String() }
+					/>
+				</div>
+			</div>
+		</div>
+		<a @click="isImageLightboxOpen = true" class="cursor-pointer">
+			<img
+				src={ screenshotUrl.String() }
+				alt={ itemName.String() + " Avatar" }
+				class="max-h-30 m-auto rounded-md"
+			/>
+		</a>
+	</div>
+}
+
+func transformMarketplaceCatalogItemScreenshotUrlsIntoCarouselImages(
+	itemFirstSlug valueObject.MarketplaceItemSlug,
+	itemName valueObject.MarketplaceItemName,
+	screenshotUrls []valueObject.Url,
+) []templ.Component {
+	carouselImages := []templ.Component{}
+	for _, screenshotUrl := range screenshotUrls {
+		carouselImage := MarketplaceCatalogItemCarouselImage(
+			itemFirstSlug, itemName, screenshotUrl,
+		)
+		carouselImages = append(carouselImages, carouselImage)
+	}
+
+	return carouselImages
+}
```

Comment:
Wow. You are creating an entire lightbox div for each image displayed? That will make your HTML gigantic. Why didn't you create like we do with Modals? Create a Lightbox on the Index page itself, hidden by default. When someone cicks on an image, you update the Alpine state that will trigger the modal to be shown with the image clicked as the source. A much simpler approach that won't make your DOM gigantic without necessity.